xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/getusersandgroups.vm

```diff
@@ -162,9 +162,16 @@ $response.setContentType("application/json")
     #end
   #end ## foreach rights object
   #set($row = {})
+  #set($userFirstName = $user.getValue('first_name'))
+  #set($userLastName = $user.getValue('last_name'))
+  #if($stringtool.isNotBlank($userFirstName) || $stringtool.isNotBlank($userLastName))
+    #set($userDisplayName = "$!{userFirstName} $!{userLastName} (${user.name})")
```

Comment:
Note : I believe it's interesting to display the username here in case the username is significantly different from the first name / last name, however it may make the displayed names a bit too long sometimes. WDYT ?

Reply:
Ideally, this column's display should be based on the standard user displayer. That's its job, after all and it would be much more consistent with other areas of the wiki.

Reply:
Indeed, updating

Reply:
@tmortagne fixed as part of [f0e088e](https://github.com/xwiki/xwiki-platform/pull/2361/commits/f0e088e35d8466ee4e0e176019998306c6ee5012). Here is how it looks : 

On a subwiki : 

Groups : 

![image](https://github.com/xwiki/xwiki-platform/assets/5938943/3a361ac5-bfc0-434b-9f43-b0da4b5157eb)

Users : 

![image](https://github.com/xwiki/xwiki-platform/assets/5938943/bc9cafae-1e72-4060-a420-5e727d21035a)

On the main wiki : 

Groups : 

![image](https://github.com/xwiki/xwiki-platform/assets/5938943/dba65805-9735-403f-842f-099efbfafe12)

Users : 

![image](https://github.com/xwiki/xwiki-platform/assets/5938943/7967580c-2f65-4f35-ac6a-96403f7614d5)

I don't think we have functional tests for that, can you confirm by any chance ?

Reply:
I thought your initial goal was to show the alias, but it looks like you did not enable this option (which is false by default from what I see in macros.vm). It's enabled in the group, for example.

Reply:
Indeed, displaying the alias gives a much nicer result, updating the PR with `showAlias` to `true` for both groups and users.

Reply:
Fixed in fcc103b6f29 ; here is how it looks : 

![image](https://github.com/xwiki/xwiki-platform/assets/5938943/4936ba34-9041-4691-a582-c5463b1c3116)

Reply:
> I don't think we have functional tests for that, can you confirm by any chance ?

Not sure, but that area is not my strong suit. If there was something I would expect it to be in xwiki-platform-administration-test-docker or xwiki-platform-distribution-flavor-test-ui.

Reply:
Oki, I've checked and did not find any tests related to the rights matrix. I'll merge and then keep an eye on the CI to see how it evolves.